content/04.results.md

```diff
@@ -112,5 +112,32 @@ A perfectly calibrated model would follow the dashed diagonal line.
 The blue line represents the predictions before calibration and the orange line shows predictions after calibration. 
 ](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/model_calibration_experiment/model_calibration.png){#fig:discriminative_model_calibration}
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Baseline Comparison
+Once our discriminator model is calibrated, we compared our model's ability to predict edges in our test set to a previosuly published baseline model [@tag:cocoscore].
```

Comment:
The discriminator provides predictions for sentences... how do you convert from sentence predictions to edge predictions?